app/src/main/java/com/islaharper/dawnofandroid/presentation/components/Widgets.kt

```diff
@@ -0,0 +1,178 @@
+package com.islaharper.dawnofandroid.presentation.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.AppBarDefaults
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.ui.theme.TOP_BAR_HEIGHT
+
+@Composable
+fun ThemeSwitcher(
+    darkTheme: Boolean = false,
+    onClick: () -> Unit
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .padding(
+                horizontal = 16.dp,
+                vertical = 10.dp
+            ),
+        horizontalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        Text(stringResource(R.string.day_mode))
+        Switch(
+            checked = darkTheme,
+            onCheckedChange = {
+                onClick()
+            }
+        )
+        Text(stringResource(R.string.night_mode))
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SearchWidget(
```

Comment:
This Composable has a lint warning "This @Composable function emits content but doesn't have a modifier parameter."
It's showing as underlined in red in my IDE, can you please check if this happening for you? You may have to do a rebuild or invalidate caches/restart in Android Studio